<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PassForge</string>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="CentralGridLayout" rowstretch="6,4" columnstretch="3,7">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetMinAndMaxSize</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="0" column="0" rowspan="2">
      <widget class="QTreeView" name="DirectoryStructureTreeView"/>
     </item>
     <item row="0" column="1">
      <widget class="QListView" name="EntryListView"/>
     </item>
     <item row="1" column="1">
      <widget class="QTabWidget" name="HeaderInfoTabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="HeaderInfoTabCredential">
        <attribute name="title">
         <string>Credential</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>521</width>
           <height>181</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="CredentialHeaderInfoLayout">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="CredentialUsernameLabel">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Username:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="CredentialNotesLabel">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Notes:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="CredentialUsernameLineEdit">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" rowspan="5">
           <widget class="QPlainTextEdit" name="CredentialNotesTextEdit">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="CredentialPasswordLabel">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Password:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="CredentialPasswordLayout" stretch="9,1">
            <item>
             <widget class="QLineEdit" name="CredentialPasswordLineEdit">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="CredentialPasswordShowButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="CredentialUrlLabel">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>URL:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLineEdit" name="CredentialUrlLineEdit">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="HeaderInfoTabCryptocurrency">
        <attribute name="title">
         <string>Cryptocurrency</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>521</width>
           <height>181</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="CryptocurrencyHeaderInfoLayout">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="CryptocurrencyNameLabel">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Cryptocurrency</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="CryptocurrencyNotesLabel">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Notes:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="CryptocurrencyNameLineEdit">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" rowspan="5">
           <widget class="QPlainTextEdit" name="CryptocurrencyNotesTextEdit">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="CryptocurrencySeedLabel">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Seed:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="CryptocurrencyPasswordLayout" stretch="9,1">
            <item>
             <widget class="QLineEdit" name="CryptocurrencySeedLineEdit">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="CryptocurrencySeedShowButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="CryptocurrencyMasterKeyLabel">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Master Private Key:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <layout class="QHBoxLayout" name="CryptocurrencyMasterKeyLayout" stretch="9,1">
            <item>
             <widget class="QLineEdit" name="CryptocurrencyMasterKeyLineEdit">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="CryptocurrencyMasterKeyShowButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="HeaderInfoTabApiKey">
        <attribute name="title">
         <string>API Key</string>
        </attribute>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>3</y>
           <width>521</width>
           <height>171</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="ApiKeyHeaderInfoLayout">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item row="4" column="0">
           <layout class="QHBoxLayout" name="ApiKeyLayout" stretch="9,1">
            <item>
             <widget class="QLineEdit" name="ApiKeyLineEdit">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ApiKeyShowButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1" rowspan="5">
           <widget class="QPlainTextEdit" name="ApiKeyNotesTextEdit">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2">
           <widget class="QLabel" name="ApiKeyUrlLabel">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>URL:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="ApiKeyLabel">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Key:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="ApiKeyNotesLabel">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Notes:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="ApiKeyUrlLineEdit">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="MainWindowMenubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="MainWindowMenubarMenuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="ActionDatabaseNew"/>
    <addaction name="ActionDatabaseOpen"/>
    <addaction name="ActionDatabaseSave"/>
    <addaction name="ActionDatabaseSaveAs"/>
    <addaction name="separator"/>
    <addaction name="ActionDatabaseSettings"/>
    <addaction name="ActionDatabaseClose"/>
    <addaction name="separator"/>
    <addaction name="ActionApplicationQuit"/>
   </widget>
   <widget class="QMenu" name="MainWindowMenubarMenuEntries">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Entries</string>
    </property>
    <addaction name="ActionEntryNew"/>
    <addaction name="ActionEntryEdit"/>
    <addaction name="ActionEntryDuplicate"/>
    <addaction name="ActionEntryDelete"/>
    <addaction name="separator"/>
    <addaction name="ActionMoveEntryUp"/>
    <addaction name="ActionMoveEntryDown"/>
    <addaction name="separator"/>
    <addaction name="ActionCopyEntryUsername"/>
    <addaction name="ActionCopyEntryPassword"/>
    <addaction name="ActionCopyEntryAttribute"/>
    <addaction name="separator"/>
    <addaction name="ActionPerfomEntryAutoType"/>
   </widget>
   <widget class="QMenu" name="MainWindowMenubarMenuGroups">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Groups</string>
    </property>
    <addaction name="ActionGroupNew"/>
    <addaction name="ActionGroupEdit"/>
    <addaction name="ActionGroupDuplicate"/>
    <addaction name="ActionGroupDelete"/>
   </widget>
   <widget class="QMenu" name="MainWindowMenubarMenuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="ActionToolsPasswordGenerator"/>
    <addaction name="ActionToolsSettings"/>
   </widget>
   <widget class="QMenu" name="MainWindowMenubarMenuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="ActionAboutPassForge"/>
   </widget>
   <addaction name="MainWindowMenubarMenuDatabase"/>
   <addaction name="MainWindowMenubarMenuEntries"/>
   <addaction name="MainWindowMenubarMenuGroups"/>
   <addaction name="MainWindowMenubarMenuTools"/>
   <addaction name="MainWindowMenubarMenuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ActionDatabaseNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="ActionDatabaseOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="ActionDatabaseSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="ActionDatabaseSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="ActionDatabaseClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="ActionDatabaseSettings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="ActionApplicationQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="ActionEntryNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="ActionEntryEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="ActionEntryDuplicate">
   <property name="text">
    <string>Duplicate</string>
   </property>
  </action>
  <action name="ActionEntryDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="ActionMoveEntryUp">
   <property name="text">
    <string>Move Up</string>
   </property>
  </action>
  <action name="ActionMoveEntryDown">
   <property name="text">
    <string>Move Down</string>
   </property>
  </action>
  <action name="ActionCopyEntryUsername">
   <property name="text">
    <string>Copy Username</string>
   </property>
  </action>
  <action name="ActionCopyEntryPassword">
   <property name="text">
    <string>Copy Password</string>
   </property>
  </action>
  <action name="ActionCopyEntryAttribute">
   <property name="text">
    <string>Copy Attribute</string>
   </property>
  </action>
  <action name="ActionPerfomEntryAutoType">
   <property name="text">
    <string>Perfom Auto-Type</string>
   </property>
  </action>
  <action name="ActionGroupNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="ActionGroupEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="ActionGroupDuplicate">
   <property name="text">
    <string>Duplicate</string>
   </property>
  </action>
  <action name="ActionGroupDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="ActionToolsPasswordGenerator">
   <property name="text">
    <string>Password Generator</string>
   </property>
  </action>
  <action name="ActionToolsSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="ActionAboutPassForge">
   <property name="text">
    <string>About PassForge</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
